kernel/arch/dreamcast/kernel/timer.c

```diff
@@ -120,35 +169,39 @@ void timer_spin_sleep(int ms) {
     timer_stop(TMU1);
 }
 
-/* Enable timer interrupts (high priority); needs to move
-   to irq.c sometime. */
+/* Enable timer interrupts; needs to move to irq.c sometime. */
 void timer_enable_ints(int which) {
-    volatile uint16 *ipra = (uint16*)0xffd00004;
-    *ipra |= (0x000f << (12 - 4 * which));
+    volatile uint16_t *ipra = (uint16_t *)0xffd00004;
```

Comment:
No define for 0xffd00004 ? :)

Reply:
You're right... I have a separate PR for cleaning up `irq.h` and have added all of these `#defines` for the priority registers there. Was going to handle it separately, in a different PR, if that's okay with everyone. Yes, it needs to be cleaned up. Here's a screenshot of my WIP local branch adding the defines:
![Screenshot from 2023-11-22 22-42-13](https://github.com/KallistiOS/KallistiOS/assets/5545520/1d38fb23-2250-4b0b-987d-27dec5714a16)